INSTALL.md

```diff
@@ -547,6 +547,28 @@ are required for the various QGIS source formatting and preparation scripts.
 dnf install ccache ninja-build astyle python3-autopep8 python3-mock python3-nose2 perl-YAML-Tiny
 ```
 
+### 3.11.4. QT6 experimental builds with Fedora Rawhide
+
+This requires latest QGIS master (>= January 25th 2024).
+
+To save ~ 700 MB, if you want to install PROJ without its optional grids, first do:
+
+```bash
+dnf5 install -y --setopt=install_weak_deps=False proj-devel
+```
+
+Installation of all needed build dependencies:
+
+```bash
+dnf5 install qt6-qtbase-private-devel qt6-qtwebkit-devel qt6-qtlocation-devel qt6-qtmultimedia-devel qt6-qttools-static qca-qt6-devel qca-qt6-ossl qt6-qt3d-devel python3-qt6-devel python3-qscintilla-qt6-devel qscintilla-qt6-devel python3-qscintilla-qt6 clang flex bison geos-devel gdal  gdal-devel hdf5-devel sqlite-devel libspatialite-devel qt6-qtsvg-devel spatialindex-devel expat-devel netcdf-devel proj-devel qwt-qt6-devel gsl-devel PDAL PDAL-devel postgresql-devel cmake python3-gdal gdal-python-tools python3-psycopg2 python3-PyYAML python3-pygments python3-jinja2 python3-OWSLib qca-qt6-ossl qwt-qt6-devel qtkeychain-qt6-devel libzip-devel exiv2-devel  PyQt-builder protobuf-lite protobuf-lite-devel libzstd-devel qt6-qtserialport-devel draco-devel python3-devel qt6-qt5compat-devel
```

Comment:
There's some missing here -- eg sip6, qt6-qtdeclarative-devel, qt6-qtpositioning-devel, qt6-qtbase-devel, ... . Was this list taken from the qt6 dockerfile?

Reply:
All of them got implicitly installed from the above apparently:
```
[root@ca0d6a9feda8 build_fedora_qt6]# rpm -qa | grep sip6
sip6-6.8.2-1.fc40.noarch
[root@ca0d6a9feda8 build_fedora_qt6]# rpm -qa | grep qt6-qtdeclarative-devel
qt6-qtdeclarative-devel-6.6.1-1.fc40.x86_64
[root@ca0d6a9feda8 build_fedora_qt6]# rpm -qa | grep qt6-qtpositioning-devel
qt6-qtpositioning-devel-6.6.1-1.fc40.x86_64
[root@ca0d6a9feda8 build_fedora_qt6]# rpm -qa | grep  qt6-qtbase-devel
qt6-qtbase-devel-6.6.1-1.fc40.x86_64
```
That list was adapted from the Fedora 38 Workstation at line 478, by mostly replacing qt5 with qt6, and adding qt6-qt5compat-devel, as I didn't manage to find a better bootstrapping source

Reply:
Ok, let's merge and then improve incrementally!